src/components/common/WhiteButton/styles.js

```diff
@@ -0,0 +1,15 @@
+import { StyleSheet } from 'react-native';
+
+export const style = (alignSelf, borderBottomColor, borderBottomWidth, marginVertical,
+  width) =>
+  StyleSheet.create({
```

Comment:
Alo! Don't use create for styles that depend on parameters.
```suggestion
  ({
```